stanford/roles/deployment/defaults/main.yml

```diff
@@ -5,4 +5,14 @@ DEPLOYMENT_NAME: "{{ COMMON_DEPLOYMENT }}-vpc"
 DEPLOYMENT_REGION: "{{ region }}"
 DEPLOYMENT_SSL_CERTIFICATE_ID: 'arn:aws:acm:<REGION>:<ACCOUNT>:certificate/<GUID>'
 # DEPLOYMENT_VPC_ID: vpc-deadbeefcafe
-DEPLOYMENT_NUMBER: "{{ cluster_subnet_index|int + 1 | int }}"
+deployment_subnet_index: "{{ (CLUSTER_NUMBER - 1 |int|abs) % 2 |int }}"
```

Comment:
Between the change here and in `stanford/roles/cluster/defaults/main.yml`,
we address a bug affecting `CLUSTER_NUMBER >= 2 or CLUSTER_NUMBER < 0`.